### PR TITLE
Fix security issues deep source

### PR DESCRIPTION
**Pull request by jLopezbarb**
# Proposed changes

this PR addresses fixes for Style issues at Deepsource master analysis.
https://deepsource.io/gh/okteto/okteto/issues?category=security&page=1

### CODE REVIEW

File: cmd/utils/git_test.go [hunk 1]
```diff
@@ -147,3 +148,9 @@ func Test_isOktetoRepoFromURL(t *testing.T) {
 		})
 	}
 }
+
+func TestGetRandomSHA(t *testing.T) {
```
Comment (AgustinRamiroDiaz) [inline cmd/utils/git_test.go:152]: We could try asserting that 2 SHAs are different

File: cmd/utils/git_test.go [hunk 2]
```diff
@@ -147,3 +148,9 @@ func Test_isOktetoRepoFromURL(t *testing.T) {
...
+	sha := GetRandomSHA()
+	assert.Len(t, sha, 40)
+	assert.NotEqual(t, sha, strings.Repeat("0", 40))
```
Comment (teresaromero) [inline cmd/utils/git_test.go:155]: can we have `strings.Repeat("0",40)` as variable?

File: cmd/utils/git.go [hunk 3]
```diff
@@ -88,11 +90,16 @@ func IsCleanDirectory(ctx context.Context, path string) (bool, error) {
...
-func GetRandomSHA(ctx context.Context, path string) string {
+// GetRandomSHA returns a random sha generated in the fly
+func GetRandomSHA() string {
 	var letters = []rune("0123456789abcdef")
 	b := make([]rune, 40)
 	for i := range b {
-		b[i] = letters[rand.Intn(len(letters))]
+		n, err := rand.Int(rand.Reader, big.NewInt(int64(len(letters))))
+		if err != nil {
+			oktetoLog.Info("could not generate new int")
```
Comment (teresaromero) [inline cmd/utils/git.go:100]: if `err != nil` what is `n` ? should we continue the for loop here? if not, will the `n.Int64()` panic?

**Review by teresaromero — APPROVED** [review]
minor comments, LGTM

File: cmd/utils/git.go (hunk 3 above)
Comment (jLopezbarb) [reply cmd/utils/git.go:100]: if `err != nil` then `n = strings.Repeat("0", 40)`

**Review by AdrianPedriza — APPROVED** [review]

**Pull request merged**